src/modules/GroupChannel/components/SuggestedReplies/index.tsx

```diff
@@ -1,15 +1,36 @@
 import './index.scss';
 import React, { useState } from 'react';
-import { BaseMessage } from '@sendbird/chat/message';
 
 export interface SuggestedRepliesProps {
   replyOptions: string[];
   onSendMessage: ({ message }: { message: string }) => void;
-  message: BaseMessage;
   type?: 'vertical' | 'horizontal';
   gap?: number;
 }
 
+export interface ReplyItemProps {
+  value: string;
+  index: number;
+  onClickReply: (event: React.MouseEvent<HTMLDivElement>, option: string) => void;
+}
+
+export const ReplyItem = ({
+  value,
+  index,
+  onClickReply,
+}: ReplyItemProps) => {
+  return (
+    <div
+      className="sendbird-suggested-replies__option"
+      id={value}
+      key={index + value}
```

Comment:
```suggestion
  onClickReply,
}: ReplyItemProps) => {
  return (
    <div
      className="sendbird-suggested-replies__option"
      id={value}
```
We don't need the key here since you pass it inside of the children below :)

Reply:
`index` can be removed from the props as well